framework/src/main/java/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -30,11 +31,17 @@ public AnnotatedTypeMirror visitVariable(VariableTree node, AnnotatedTypeFactory
         AnnotatedTypeMirror result = TypeFromTree.fromTypeTree(f, node.getType());
 
         // Add primary annotations
```

Comment:
Can you add a comment to the method, similar to what org.checkerframework.framework.util.element.ElementAnnotationUtil.addAnnotationsFromElement is doing?
Cross-references between the two places are probably also a good idea, to make sure they get updated together.

Reply:
Done below.